release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2478,6 +2478,145 @@ index a90ab14ff..f7f6f1a71 100644
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=2097579[*BZ#2097579*])
 
+[id="ocp-4-11-ran-known-issues"]
+== {product-title} 4.11 RAN known issues
+
+* After deploying a {sno} with the RAN DU profile, an error `openvswitch: cpu_id mismatch with handler threads` is continuously generated in the Open vSwitch kernel logs. (link:https://issues.redhat.com/browse/OCPBUGSM-46165[*OCPBUGSM-46165*])
+
+* If secure boot is currently disabled and you attempt to enable it through ZTP, the cluster installation does not start. When secure boot is enabled through ZTP, the boot options are configured before the virtual CD is attached. Therefore, the first boot is from the existing hard disk with secure boot turned on. The cluster installation gets stuck because the system never boots from the CD. (link:https://issues.redhat.com/browse/OCPBUGSM-45085[*OCPBUGSM-45085*])
+
+* If an invalid subscription channel is specified in the subscription policy that is used to perform a cluster upgrade, the {cgu-operator-full} indicates a successful upgrade right after the policy is enforced because the `Subscription` state remains `AtLatestKnown`. (link:https://issues.redhat.com/browse/OCPBUGSM-43618[*OCPBUGSM-43618*])
+
+* If the SRIOV-FEC Operator is installed through separate policies and the `CatalogSource` reuses the name of a default catalog source, the Operator installation may fail due to conflicts with the management of the default catalog sources. To avoid this issue the SRIOV-FEC `CatalogSource` CR should be added to the `common-config-policy` and the operator subscription should be added to the `common-subscriptions-policy`. As a workaround when using separate policies to install the SRIOV-FEC Operator, you must ensure that the `CatalogSource` for this Operator is uniquely named. (link:https://issues.redhat.com/browse/OCPBUGSM-39859[*OCPBUGSM-39859*])
```

Comment:
```suggestion
* If the SRIOV-FEC Operator is installed through separate policies and the `CatalogSource` reuses the name of a default catalog source, the Operator installation might fail due to conflicts with the management of the default catalog sources. To avoid this issue the SRIOV-FEC `CatalogSource` CR should be added to the `common-config-policy` and the operator subscription should be added to the `common-subscriptions-policy`. As a workaround when using separate policies to install the SRIOV-FEC Operator, you must ensure that the `CatalogSource` for this Operator is uniquely named. (link:https://issues.redhat.com/browse/OCPBUGSM-39859[*OCPBUGSM-39859*])
```

Reply:
See the ISG entry for 'may': https://www.ibm.com/docs/en/ibm-style?topic=word-usage